docs/joss_paper/paper.md

```diff
@@ -82,35 +82,35 @@ The loss functions described in the previous sections are often combined with a
 The latest result from a different registration formulation challenged the use of deformation regularisation and consequently the necessity of predicting dense displacement field [@hu2019conditional], also implemented in DeepReg.
 
 # Implemetation
-## Model
-A pair of moving image and fixed image is the input of the network. When available, pairs of moving labels and/or fixed labels can also be the input of the network, regardless of whether they are used to compute the loss. The network predicts a dense displacement field (DDF). Options are also available for the DDF being integrated by a predicted dense velocity field (DFV) or computed from a predicted affine transformation. For conditional segmentation, the network directly predicts warped moving labels [@hu2019conditional].
+## Registration network
+A pair of moving image and fixed image is the input of the network. When available, pairs of moving labels and/or fixed labels can also be the input of the network to compute the label dissimilarity loss. The network predicts a dense displacement field (DDF). Options are also available for the DDF being integrated by a predicted dense static velocity field, often as SVF or DVF, or computed from a predicted affine transformation. For conditional segmentation, the network directly predicts warped moving labels [@hu2019conditional].
 
-An encoder-decoder architecture, such as an adapted U-Net and the local-net [@hu2018weakly] is used for dense prediction such as direct DDF or DVF. An encoder-only architecture, such as ResNet and VGG, is used to predict the parameters of an affine transformation model, with 12 degrees of freedom.
+An encoder-decoder architecture, such as an adapted U-Net and the local-net [@hu2018weakly], is used for dense prediction such as DDF or DVF. An encoder-only architecture, such as ResNet and VGG, is used to predict the parameters of an affine transformation model, with 12 degrees of freedom.
 
-## Model training
-Registration networks are trained using standard minibatch gradient descent, with each minibatch sampling pairs of moving-fixed images and, when available, moving-and-fixed labels.
+## Model training and prediction
+Registration networks are trained using standard minibatch gradient descent, with each minibatch sampling pairs of moving- and fixed images. Optionally, when available, a pair of moving- and fixed labels.
```

Comment:
"Optionally, when available, a pair of moving- and fixed labels." -> "Optionally, when available, a pair of moving- and fixed labels are used during training."